src/blueprints/tv2/action-factories/tv2-audio-action-factory.ts

```diff
@@ -22,15 +22,18 @@ import { Tv2PieceType } from '../enums/tv2-piece-type'
 import {
   Tv2VideoClipTimelineObjectFactory
 } from '../timeline-object-factories/interfaces/tv2-video-clip-timeline-object-factory'
+import { ActionFactory } from './ActionFactory'
 
 const FRAME_RATE: number = 25
 
-export class Tv2AudioActionFactory {
+export class Tv2AudioActionFactory extends ActionFactory {
```

Comment:
What is the reason for using inheritance over composition here? :)

Reply:
Less dependencies to inject. I don't want to end up in a place where we have to inject tons of "helper" classes.